### ISSUE TITLE
Update article on eager execution on website

### ISSUE THREAD

**Issue by mpettis** [rstudio/keras#1010]
I installed keras, which now gets tensorflow v2, and I tried following the documentation on eager execution here (https://keras.rstudio.com/articles/eager_guide.html) because I want to be able to use GradientTape an control `apply_gradient()` for some keras examples I'd like to translate from Geron's "Hands-On Machine Learning with Scikit-Learn, Keras, and TensorFlow, 2nd Edition".

I tried running the code here on eager execution guide here (https://keras.rstudio.com/articles/eager_guide.html), but I believe it it written for tensorflow v1.  I had to make two tweaks, but I was able to make it work with the default keras install.

Can/should this be rewritten for tensorflow v2?  As I can't seem to find any git repo for this particular code, I'll inline changes to that code I made:

```
#tfe_enable_eager_execution(device_policy = "silent")
```
```
#optimizer <- tf$train$AdamOptimizer()
optimizer <- tf$optimizers$Adam()
```

```
            optimizer$apply_gradients(purrr::transpose(list(gradients, model$variables))
                                      #global_step = tf$train$get_or_create_global_step())
                                      )

```

**Comment by henry090** [comment]
Hi, could you share the python code which you would like to translate? So, we could help.

**Comment by mpettis** [comment]
Sure, thanks!  It is at this link: https://github.com/ageron/handson-ml2/blob/master/18_reinforcement_learning.ipynb , about 1/3 of the way down... the code in the `Policy Gradients` section.  The function `play_one_step()` can be searched for -- it is the first code chunk of the relevant section.

**Comment by henry090** [comment]
I think, ```optimizer``` should be defined from ```tf$keras```:
```
optimizer <- tf$keras$optimizers$Adam()
```
And it works fine. I have just tested.

> Note: For now please use only Tensorflow 2/2.1. **Not nightly release**.

**Comment by mpettis** [comment]
Confirmed, it work for me to change it to your suggestion as well.  Thanks.

**Comment by skeydan** [comment]
@turgut thanks!

@mpettis please switch to using the documentation at https://tensorflow.rstudio.com/. All the code there works with TensorFlow 2, and there are a lot more new examples than on the "old" Keras website. Thanks!
